test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -39,13 +34,15 @@ const (
 	threshold60K = 30
 )
 
+
 // TestSchedule100Node3KPods schedules 3k pods on 100 nodes.
 func TestSchedule100Node3KPods(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping because we want to run short tests")
 	}
-
-	config := defaultSchedulerBenchmarkConfig(100, 3000)
+	//config := defaultSchedulerBenchmarkConfig(inputConfig.numNodes, inputConfig.numPods)
+	config := baseConfig()
```

Comment:
I am just adding them in the function directly. If you want I can change the method to have those variables there.